common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -714,6 +717,13 @@ def _parse_video_xml(cls, xml, id_generator=None):
                 # for about a month we did it for Strings also).
                 field_data[attr] = deserialize_field(cls.fields[attr], value)
 
+        # Update the handout location with current course_id
+        if 'handout' in field_data.keys() and course_id:
+            handout_location = StaticContent.get_location_from_path(field_data['handout'])
+            if isinstance(handout_location, AssetLocator):
```

Comment:
when its not ```AssetLocator``` we does not to process that and test for this is already present at 
https://github.com/edx/edx-platform/blob/dcc86a3b1b4f03bc84da8e78e063b1ef553714cc/common/lib/xmodule/xmodule/tests/test_video.py#L281